exercises/meetup/meetup.js

```diff
@@ -0,0 +1,8 @@
+//
+// This is only a SKELETON file for the 'Meetup' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+function getDayIndex(day) {
```

Comment:
Tests expect function with name `meetupDay` to be exported from solution file: https://github.com/exercism/javascript/blob/6cdbb89e5bcd4d0c43bc676ce5871ca814589399/exercises/meetup/meetup.spec.js#L1

Can you rename this function?

Reply:
Thanks for the feedback.  I will refactor and submit an updated PR.